src/System.Threading.ThreadPool/ref/System.Threading.ThreadPool.csproj

```diff
@@ -2,14 +2,16 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <OutputType>Library</OutputType>
     <NuGetTargetMoniker>.NETStandard,Version=v1.7</NuGetTargetMoniker>
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Threading.ThreadPool.cs" />
+    <ProjectReference Include="..\..\System.Threading.Overlapped\ref\System.Threading.Overlapped.csproj" />
```

Comment:
>ProjectReference [](start = 5, length = 16)

Can you please add a comment above this line saying that we need to remove this once packages are updated? we use this comments to find and replace all of the places we have temporary P2P references. Also, I would have expected that you would need a pkgproj reference on the actual implementation of ThreadPool to Overlapped.